pkg/embeddedcluster/oras_test.go

```diff
@@ -0,0 +1,38 @@
+package embeddedcluster
+
+import (
+	"context"
+	"testing"
+)
+
+func x_Test_pullArtifact(t *testing.T) {
+	type args struct {
+		ctx     context.Context
+		srcRepo string
+		opts    pullArtifactOptions
+	}
+	tests := []struct {
+		name    string
+		args    args
+		wantErr bool
+	}{
+		{
+			name: "test1",
+			args: args{
+				ctx:     context.Background(),
+				srcRepo: "ttl.sh/ethan/embedded-cluster-operator-bin:24h",
```

Comment:
is this what we want to use for testing?

Reply:
the test is prefixed with an x_ so it does not run. do you want me to remove it?

Reply:
if it does not run, then we should probably just remove it yeah.